SPECS-EXTENDED/perl-Class-Method-Modifiers/perl-Class-Method-Modifiers.spec

```diff
@@ -32,19 +32,14 @@ BuildRequires:  perl(Exporter)
 # Test Suite
 BuildRequires:  perl(File::Spec)
 BuildRequires:  perl(if)
-BuildRequires:  perl(Test::Fatal)
 BuildRequires:  perl(Test::More) >= 0.88
-BuildRequires:  perl(Test::Needs)
 # Optional Test Requirements
-%if 0%{!?perl_bootstrap:1} && %{with perl_Class_Method_Modifiers_enables_optional_test}
```

Comment:
It seems this macro is useful and needed. Please add it back.

Reply:
Updated.